### ISSUE TITLE
Iconphoto method inside root's block is undefined.

### ISSUE THREAD

**Issue by vin1antme** [AndyObtiva/glimmer-dsl-tk#5]
Hello,

Using *iconphoto* method inside root's block returns undefined method error (Error => `rescue in method_missing': undefined method `iconphoto' for main:Object (NoMethodError)).

Here is example returning error:
```
require "glimmer-dsl-tk"
include Glimmer
root {
  title 'Title'
  iconphoto TkPhotoImage.new('file' => 'form.png') # Result => Error `rescue in method_missing': undefined method `iconphoto' for main:Object (NoMethodError)
}.open
```

While with explicit receiver works without error:

```
root {
  title 'Title'
  root.iconphoto TkPhotoImage.new('file' => 'form.png')
}.open
```

And with regular Tk syntax works fine:

```
require 'tk'
root = TkRoot.new {
  title "Feet to Meters"
  iconphoto TkPhotoImage.new('file' => 'form.png')
}
Tk.mainloop()
```

**Expected behavior** is like in the first example  - *iconphoto* is option of root and should be defined inside root.

**Comment by AndyObtiva** [comment]
[Fixed](https://rubygems.org/gems/glimmer-dsl-tk/versions/0.0.8)!

You should have an even shorter alternative now with smart default/convention accepting image path directly:

```ruby
root {
  title 'Title'
  iconphoto 'form.png'
}.open
```

This other example works for me in `bin/girb` under cloned project:

```ruby
root {
  title 'Title'
  iconphoto 'images/glimmer-dsl-tk-screenshot-sample-hello-world.png'
}.open
```

Also, [documented](https://github.com/AndyObtiva/glimmer-dsl-tk#smart-defaults-and-convensions)!

Cheers!

Andy Maleh

p.s. Thanks for reporting! Please keep in mind this project is still an early alpha proof-of-concept experiment and most of my time is directed at [Glimmer DSL for SWT](https://github.com/AndyObtiva/glimmer-dsl-swt), among other projects.

**Comment by vin1antme** [comment]
Thank you very much for such a quick fix!

Just to let you know - there is one more small issue appeared .
While running glimmer-dsl-tk sample application there is redundant output in console:
 ```invalid log level:```

Looks like the source of that issue is inside **glimmer-2.1.1** package in that file  *\lib\glimmer\config.rb* and that variable - *ENV['GLIMMER_LOGGER_LEVEL']*

**Comment by AndyObtiva** [comment]
Fair enough. It's a false negative. Thanks for reporting. Fixed in [Glimmer 2.1.2](https://rubygems.org/gems/glimmer/versions/2.1.2) and [Glimmer DSL for Tk 0.0.9](https://rubygems.org/gems/glimmer-dsl-tk/versions/0.0.9)
